sec_certs_page/templates/cc/entry/index.html.jinja2

```diff
@@ -446,6 +446,11 @@
                 {% endif %}
             </div>
 
+            {% if additional_metadata %}
+                <h3 class="mt-3">Additional Metadata</h3>
+                {{ metadata_binding_card("cc", additional_metadata["metadata_headers"]) }}
```

Comment:
The first argument `doc_prefix` is not meant to be "cc" or "fips" but either `"cert"` or `"target"` based on whether the card is inside the "certification report" part of the page or the "security target" part. In this case the card is part of neither, so the doc prefix need not be used, instead something like "metadata" should be used to make the element ids unique.